apps/system-e2e/src/support/session.ts

```diff
@@ -90,11 +91,11 @@ async function ensureIDSsession(
 
 export async function session({
   browser,
-  storageState,
   homeUrl,
   phoneNumber,
   authUrl,
   idsLoginOn,
+  storageState = `${homeUrl}-${phoneNumber}`,
```

Comment:
Nice!

We might want to add the delegationNationalId parameter here as well, and include it in the default storageState.

Also, do you need to escape homeUrl somehow since it might contain "/", and if I understand this right, it is used to create files on the filesystem.

Reply:
Added delegation to the default

Reply:
If a `storageState` contains `/`, new directories are created. This doesn't matter, since the lookup resolves into the directory. I.e. functionally, there is no difference.

There is no mention in the [Playwright docs](https://playwright.dev/docs/api/class-browsercontext#browser-context-storage-state) on whether the directories are created on save. However, in my testing, sub-directories are created and the browser context is indeed stored as expected.

Reply:
Mainly mentioning this because homeUrl usually starts with /, like "/stjornbord". Depending on the fs logic it could cause problems ("folder//file", or resolved to a filesystem root folder because of path.resolve). 

The docs say this: "If path is a relative path, then it is resolved relative to current working directory.". And you're basically passing in an absolute path, which could cause issues now or later, ie trying to create the sessionStore in a root folder on your drive.

Reply:
Good catch. Made some changes in https://github.com/island-is/island.is/pull/9515/commits/f584511fabcd322e98299ab56b18e672647180e4 to counteract this. "Paths" like `folder//file` should resolve to `folder/file`.

Using folders gives clear scoping of who is doing what.